pom.xml

```diff
@@ -3,7 +3,7 @@
 
   <groupId>org.folio</groupId>
   <artifactId>data-import-processing-core</artifactId>
-  <version>1.1.0-SNAPSHOT</version>
+  <version>2.1.0-SNAPSHOT</version>
```

Comment:
I would rather bump the version on release, we don't need to bump snapshot unless we don't want this change applied in dependencies on 1.1.0-SNAPHSOT (and it is my understanding we don't have dependencies on snapshot)

Reply:
if you guys - @OleksiiKuzminov, @RuslanLavrov - disagree and insist on bumping the snapshot version, then at least bump it 2.0.0-SNAPSHOT

Reply:
@KaterynaSenchenko Thx for this comment! I forget changed minor version to 0.